packages/client/wallets/aa/src/types/Config.ts

```diff
@@ -6,12 +6,17 @@ export type CrossmintAASDKInitParams = {
     clientSecret: string;
 };
 
-export type UserIdentifier = {
-    email: string;
+export type UserIdentifierParams = {
+    email?: string;
     userId?: string;
     phoneNumber?: string;
 };
 
+export type UserIdentifier =
+    | { type: "white-label"; userId: string }
```

Comment:
nit: consistency w/ `phoneNumber`
```suggestion
    | { type: "whiteLabel"; userId: string }
```